src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -38,7 +38,16 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
             {
                 Debug.Assert(output != null);
                 Debug.Assert(profile != null);
-                Console.Clear();
+
+                try
+                {
+                    Console.Clear();
+                }
+                catch (IOException)
+                {
+                    // Console not available.
```

Comment:
nit: Should we add a link to documentation here, or explanation?

Reply:
Fixed in 2nd iteration